api/controllers/game/ready.js

```diff
@@ -28,7 +28,7 @@ module.exports = function (req, res) {
         }
         if (bothReady) {
           // Inform all clients this game is starting
```

Comment:
```suggestion
          // Inform all clients this game has started
```
Stick with your new case. I usually prefer the now tense, but I see why you've don it this way instead. As long as we're consistent I don't mind either way.